azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/components/JoinCallButtonHolderView.kt

```diff
@@ -92,4 +98,9 @@ internal class JoinCallButtonHolderView : ConstraintLayout {
             joiningCallText.visibility = GONE
         }
     }
+
+    private fun isMicrophoneAvailable(): Boolean {
```

Comment:
Name isn't aligned with action, should be
isAudioModeNormal()
or
isAudioModeNotInCommunication()

This isn't actually a mic check

Reply:
So it is another way of checking the mic availability. The recording check approach does not work as it records even if the mic is acquired by other apps in latest Android version. I'll rename and put the intention in the method as doc. 
Thanks for calling this out.

Reply:
So it is another way of checking the mic availability. The recording check approach does not work as it records even if the mic is acquired by other apps in latest Android version. I'll rename and put the intention in the method as doc. 
Thanks for calling this out.